docs/how-to-guides/replication.md

```diff
@@ -105,10 +105,11 @@ instances:
    REPLICA instance 1 is called REP1, its replication mode is SYNC, and it is
    located at IP address `172.17.0.3.` with port `10000`.
 
-   When the REPLICA instance is running in SYNC mode, the MAIN instance will not
-   commit a transaction until all REPLICA instances running in the SYNC mode
-   confirm they have received the same transaction. SYNC mode prioritizes data
-   consistency but has no tolerance for any network failures.
+   When the REPLICA instance is running in SYNC mode, once the MAIN instance commits a transaction, it will
```

Comment:
```suggestion
   Once the MAIN instance commits a transaction, it will
```

Reply:
I think this is not necessary as we mention "until all REPLICA instances running in the SYNC mode
   confirm they have received the same transaction." later in the sentence